richreview/public/webapps/richreview/r2/model/dom_model.js

```diff
@@ -49,9 +49,119 @@
 
         pub.remove = function(annot_id){
             var annot_id_esc = r2.util.escapeDomId(annot_id);
+            var parentPiece=$('#'+annot_id_esc).parent('.tc_piece');
```

Comment:
Could you name variables with under-bar separation, and function names with Camel cases? That's the convention that I've used throughout this repository.